.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,93 @@
+name: "Bug report"
+description: "Create a report to help us improve TypeScript"
+title: "[BUG] <description>"
```

Comment:
I don't know if we should have `[BUG]` in the title of all bugs; can we just leave this blank?